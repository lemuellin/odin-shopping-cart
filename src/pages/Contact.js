import '../style/contact.css';
import facebook from '../asset/contact/facebook.png';
import github from '../asset/contact/github.png';
import instagram from '../asset/contact/instagram.png';
import linkedin from '../asset/contact/linkedin.png';
import messenger from '../asset/contact/messenger.png';
import twitter from '../asset/contact/twitter.png';

const Contact = () => {
  return (
    <div className='contact'>
      <div className='link'>
        <a href='https://github.com/lemuellin' target="_blank" rel="noopener noreferrer">
          <img src={github} alt='github' className='icon'/>
        </a>
        <a href='https://www.linkedin.com/in/lemuel-lin/' target="_blank" rel="noopener noreferrer">
          <img src={linkedin} alt='linkedin' className='icon'/>
        </a>
        <a href='https://www.youtube.com/shorts/AWOyEIuVzzQ?&ab_channel=UFCHub' target="_blank" rel="noopener noreferrer">
          <img src={facebook} alt='facebook' className='icon'/>
        </a>
        <a href='https://www.youtube.com/shorts/lKz1fr18GE4?&ab_channel=Btfun' target="_blank" rel="noopener noreferrer">
          <img src={messenger} alt='messenger' className='icon'/>
        </a>
        <a href='https://www.youtube.com/watch?v=H9K8-3PHZOU&ab_channel=HamptonTheHampster-Topic' target="_blank" rel="noopener noreferrer">
          <img src={instagram} alt='instagram' className='icon'/>
        </a>
        <a href='https://www.youtube.com/watch?v=CmIKR458M0A&ab_channel=VeggieTalesOfficial' target="_blank" rel="noopener noreferrer">
          <img src={twitter} alt='twitter' className='icon'/>
        </a>
      </div>
      <div className='address'>
        <h2>Minimal Garden</h2>
        <h2>Los Angeles, CA</h2>
      </div>
    </div>
  );
}
  
  export default Contact;