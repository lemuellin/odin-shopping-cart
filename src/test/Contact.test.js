import React from "react";
import { render } from "@testing-library/react";
import Contact from '../pages/Contact';

describe("Contact", () => {
    it("Match Snapshot", () => {
        const { container } = render (<Contact/>);
        expect(container).toMatchSnapshot();
    });

});