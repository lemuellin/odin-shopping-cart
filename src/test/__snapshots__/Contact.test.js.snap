// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Match Snapshot 1`] = `
<div>
  <div
    class="contact"
  >
    <div
      class="link"
    >
      <a
        href="https://github.com/lemuellin"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="github"
          class="icon"
          src="github.png"
        />
      </a>
      <a
        href="https://www.linkedin.com/in/lemuel-lin/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="linkedin"
          class="icon"
          src="linkedin.png"
        />
      </a>
      <a
        href="https://www.youtube.com/shorts/AWOyEIuVzzQ?&ab_channel=UFCHub"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="facebook"
          class="icon"
          src="facebook.png"
        />
      </a>
      <a
        href="https://www.youtube.com/shorts/lKz1fr18GE4?&ab_channel=Btfun"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="messenger"
          class="icon"
          src="messenger.png"
        />
      </a>
      <a
        href="https://www.youtube.com/watch?v=H9K8-3PHZOU&ab_channel=HamptonTheHampster-Topic"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="instagram"
          class="icon"
          src="instagram.png"
        />
      </a>
      <a
        href="https://www.youtube.com/watch?v=CmIKR458M0A&ab_channel=VeggieTalesOfficial"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="twitter"
          class="icon"
          src="twitter.png"
        />
      </a>
    </div>
    <div
      class="address"
    >
      <h2>
        Minimal Garden
      </h2>
      <h2>
        Los Angeles, CA
      </h2>
    </div>
  </div>
</div>
`;
